src/js/forms.js

```diff
@@ -134,6 +135,59 @@ class Forms {
 		this.FormEl = undefined;
 	}
 
+	static updateInputStatus (input, status) {
+		if (!(input instanceof HTMLElement)) {
+			throw new TypeError(`Expecting "input" to be an instance of "HTMLElement" but instead found ${oFormsEl instanceof HTMLElement}.`);
```

Comment:
For any input which is not an HTMLElement instance, this would error with:
>Expecting "input" to be an instance of "HTMLElement" but instead found false.

The "but instead found false" could make a user think that they passed in `false`, which they may not have done. 
Perhaps something like:
>`Expected input (${input}) to be of type "HTMLElement".